{"text": "2024-05-29 06:19:41 | INFO    | __main__:stash:131 | run - {'image': '20231121', 'version': '0.10.1.post2', 'role': 'EpicPlayer', 'headless': True}\n", "record": {"elapsed": {"repr": "0:00:01.601581", "seconds": 1.601581}, "exception": null, "extra": {"image": "20231121", "version": "0.10.1.post2", "role": "EpicPlayer", "headless": true}, "file": {"name": "claim.py", "path": "/home/runner/work/epicfree/epicfree/epic/src/claim.py"}, "function": "stash", "level": {"icon": "‚ÑπÔ∏è", "name": "INFO", "no": 20}, "line": 131, "message": "run", "module": "claim", "name": "__main__", "process": {"id": 3658, "name": "MainProcess"}, "thread": {"id": 139761568004992, "name": "MainThread"}, "time": {"repr": "2024-05-29 06:19:41.017015+08:00", "timestamp": 1716934781.017015}}}
{"text": "2024-05-29 06:19:50 | INFO    | __main__:claim_epic_games:97 | Try to flush cookie - {'task': 'claim_epic_games'}\n", "record": {"elapsed": {"repr": "0:00:10.608132", "seconds": 10.608132}, "exception": null, "extra": {"task": "claim_epic_games"}, "file": {"name": "claim.py", "path": "/home/runner/work/epicfree/epicfree/epic/src/claim.py"}, "function": "claim_epic_games", "level": {"icon": "‚ÑπÔ∏è", "name": "INFO", "no": 20}, "line": 97, "message": "Try to flush cookie", "module": "claim", "name": "__main__", "process": {"id": 3658, "name": "MainProcess"}, "thread": {"id": 139761568004992, "name": "MainThread"}, "time": {"repr": "2024-05-29 06:19:50.023566+08:00", "timestamp": 1716934790.023566}}}
{"text": "2024-05-29 06:19:53 | INFO    | epic_games.agent:_login:237 | login-with-email - {'url': 'https://www.epicgames.com/id/login?lang=en-US&noHostRedirect=true&redirectUrl=https://store.epicgames.com/en-US/free-games'}\n", "record": {"elapsed": {"repr": "0:00:13.601037", "seconds": 13.601037}, "exception": null, "extra": {"url": "https://www.epicgames.com/id/login?lang=en-US&noHostRedirect=true&redirectUrl=https://store.epicgames.com/en-US/free-games"}, "file": {"name": "agent.py", "path": "/home/runner/work/epicfree/epicfree/epic/src/epic_games/agent.py"}, "function": "_login", "level": {"icon": "‚ÑπÔ∏è", "name": "INFO", "no": 20}, "line": 237, "message": "login-with-email", "module": "agent", "name": "epic_games.agent", "process": {"id": 3658, "name": "MainProcess"}, "thread": {"id": 139761568004992, "name": "MainThread"}, "time": {"repr": "2024-05-29 06:19:53.016471+08:00", "timestamp": 1716934793.016471}}}
{"text": "2024-05-29 06:19:54 | ERROR    | hcaptcha_challenger.agents.playwright.control:handler:153 | 'utf-8' codec can't decode byte 0x9b in position 4: invalid start byte - {}\nTraceback (most recent call last):\n  File \"/home/runner/work/epicfree/epicfree/epic/src/claim.py\", line 162, in <module>\n    asyncio.run(run())\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/runners.py\", line 44, in run\n    return loop.run_until_complete(main)\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 636, in run_until_complete\n    self.run_forever()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 603, in run_forever\n    self._run_once()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 1909, in _run_once\n    handle._run()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/events.py\", line 80, in _run\n    self._context.run(self._callback, *self._args)\n> File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py\", line 145, in handler\n    data = await response.json()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/playwright/async_api/_generated.py\", line 614, in json\n    return mapping.from_maybe_impl(await self._impl_obj.json())\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/playwright/_impl/_network.py\", line 613, in json\n    return json.loads(await self.text())\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/playwright/_impl/_network.py\", line 610, in text\n    return content.decode()\nUnicodeDecodeError: 'utf-8' codec can't decode byte 0x9b in position 4: invalid start byte\n", "record": {"elapsed": {"repr": "0:00:15.334239", "seconds": 15.334239}, "exception": {"type": "UnicodeDecodeError", "value": "'utf-8' codec can't decode byte 0x9b in position 4: invalid start byte", "traceback": true}, "extra": {}, "file": {"name": "control.py", "path": "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py"}, "function": "handler", "level": {"icon": "‚ùå", "name": "ERROR", "no": 40}, "line": 153, "message": "'utf-8' codec can't decode byte 0x9b in position 4: invalid start byte", "module": "control", "name": "hcaptcha_challenger.agents.playwright.control", "process": {"id": 3658, "name": "MainProcess"}, "thread": {"id": 139761568004992, "name": "MainThread"}, "time": {"repr": "2024-05-29 06:19:54.749673+08:00", "timestamp": 1716934794.749673}}}
{"text": "2024-05-29 06:19:57 | DEBUG    | epic_games.agent:insert_challenge:215 | Attack challenge - {'stage': 'email_exists_prod'}\n", "record": {"elapsed": {"repr": "0:00:17.585656", "seconds": 17.585656}, "exception": null, "extra": {"stage": "email_exists_prod"}, "file": {"name": "agent.py", "path": "/home/runner/work/epicfree/epicfree/epic/src/epic_games/agent.py"}, "function": "insert_challenge", "level": {"icon": "üêû", "name": "DEBUG", "no": 10}, "line": 215, "message": "Attack challenge", "module": "agent", "name": "epic_games.agent", "process": {"id": 3658, "name": "MainProcess"}, "thread": {"id": 139761568004992, "name": "MainThread"}, "time": {"repr": "2024-05-29 06:19:57.001090+08:00", "timestamp": 1716934797.00109}}}
{"text": "2024-05-29 06:20:26 | ERROR    | hcaptcha_challenger.agents.playwright.control:execute:615 | task interrupt <backcall> - {'reason': 'Failed to obtain challenge task'}\n", "record": {"elapsed": {"repr": "0:00:47.135495", "seconds": 47.135495}, "exception": null, "extra": {"reason": "Failed to obtain challenge task"}, "file": {"name": "control.py", "path": "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py"}, "function": "execute", "level": {"icon": "‚ùå", "name": "ERROR", "no": 40}, "line": 615, "message": "task interrupt <backcall>", "module": "control", "name": "hcaptcha_challenger.agents.playwright.control", "process": {"id": 3658, "name": "MainProcess"}, "thread": {"id": 139761568004992, "name": "MainThread"}, "time": {"repr": "2024-05-29 06:20:26.550929+08:00", "timestamp": 1716934826.550929}}}
{"text": "2024-05-29 06:20:26 | DEBUG    | epic_games.agent:insert_challenge:218 | Parse result - {'stage': 'email_exists_prod', 'result': <Status.CHALLENGE_BACKCALL: 'backcall'>}\n", "record": {"elapsed": {"repr": "0:00:47.136147", "seconds": 47.136147}, "exception": null, "extra": {"stage": "email_exists_prod", "result": "backcall"}, "file": {"name": "agent.py", "path": "/home/runner/work/epicfree/epicfree/epic/src/epic_games/agent.py"}, "function": "insert_challenge", "level": {"icon": "üêû", "name": "DEBUG", "no": 10}, "line": 218, "message": "Parse result", "module": "agent", "name": "epic_games.agent", "process": {"id": 3658, "name": "MainProcess"}, "thread": {"id": 139761568004992, "name": "MainThread"}, "time": {"repr": "2024-05-29 06:20:26.551581+08:00", "timestamp": 1716934826.551581}}}
{"text": "2024-05-29 06:20:57 | WARNING    | epic_games.agent:authorize:266 | ÊâßË°åË∂ÖÊó∂ - {'task': 'authorize', 'retry': 0}\n", "record": {"elapsed": {"repr": "0:01:18.235359", "seconds": 78.235359}, "exception": null, "extra": {"task": "authorize", "retry": 0}, "file": {"name": "agent.py", "path": "/home/runner/work/epicfree/epicfree/epic/src/epic_games/agent.py"}, "function": "authorize", "level": {"icon": "‚ö†Ô∏è", "name": "WARNING", "no": 30}, "line": 266, "message": "ÊâßË°åË∂ÖÊó∂", "module": "agent", "name": "epic_games.agent", "process": {"id": 3658, "name": "MainProcess"}, "thread": {"id": 139761568004992, "name": "MainThread"}, "time": {"repr": "2024-05-29 06:20:57.650793+08:00", "timestamp": 1716934857.650793}}}
{"text": "2024-05-29 06:21:00 | INFO    | epic_games.agent:_login:237 | login-with-email - {'url': 'https://www.epicgames.com/id/login?lang=en-US&noHostRedirect=true&redirectUrl=https://store.epicgames.com/en-US/free-games'}\n", "record": {"elapsed": {"repr": "0:01:21.306969", "seconds": 81.306969}, "exception": null, "extra": {"url": "https://www.epicgames.com/id/login?lang=en-US&noHostRedirect=true&redirectUrl=https://store.epicgames.com/en-US/free-games"}, "file": {"name": "agent.py", "path": "/home/runner/work/epicfree/epicfree/epic/src/epic_games/agent.py"}, "function": "_login", "level": {"icon": "‚ÑπÔ∏è", "name": "INFO", "no": 20}, "line": 237, "message": "login-with-email", "module": "agent", "name": "epic_games.agent", "process": {"id": 3658, "name": "MainProcess"}, "thread": {"id": 139761568004992, "name": "MainThread"}, "time": {"repr": "2024-05-29 06:21:00.722403+08:00", "timestamp": 1716934860.722403}}}
{"text": "2024-05-29 06:21:02 | ERROR    | hcaptcha_challenger.agents.playwright.control:handler:153 | 'utf-8' codec can't decode byte 0xc3 in position 0: invalid continuation byte - {}\nTraceback (most recent call last):\n  File \"/home/runner/work/epicfree/epicfree/epic/src/claim.py\", line 162, in <module>\n    asyncio.run(run())\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/runners.py\", line 44, in run\n    return loop.run_until_complete(main)\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 636, in run_until_complete\n    self.run_forever()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 603, in run_forever\n    self._run_once()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 1909, in _run_once\n    handle._run()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/events.py\", line 80, in _run\n    self._context.run(self._callback, *self._args)\n> File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py\", line 145, in handler\n    data = await response.json()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/playwright/async_api/_generated.py\", line 614, in json\n    return mapping.from_maybe_impl(await self._impl_obj.json())\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/playwright/_impl/_network.py\", line 613, in json\n    return json.loads(await self.text())\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/playwright/_impl/_network.py\", line 610, in text\n    return content.decode()\nUnicodeDecodeError: 'utf-8' codec can't decode byte 0xc3 in position 0: invalid continuation byte\n", "record": {"elapsed": {"repr": "0:01:22.977375", "seconds": 82.977375}, "exception": {"type": "UnicodeDecodeError", "value": "'utf-8' codec can't decode byte 0xc3 in position 0: invalid continuation byte", "traceback": true}, "extra": {}, "file": {"name": "control.py", "path": "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py"}, "function": "handler", "level": {"icon": "‚ùå", "name": "ERROR", "no": 40}, "line": 153, "message": "'utf-8' codec can't decode byte 0xc3 in position 0: invalid continuation byte", "module": "control", "name": "hcaptcha_challenger.agents.playwright.control", "process": {"id": 3658, "name": "MainProcess"}, "thread": {"id": 139761568004992, "name": "MainThread"}, "time": {"repr": "2024-05-29 06:21:02.392809+08:00", "timestamp": 1716934862.392809}}}
{"text": "2024-05-29 06:21:04 | DEBUG    | epic_games.agent:insert_challenge:215 | Attack challenge - {'stage': 'email_exists_prod'}\n", "record": {"elapsed": {"repr": "0:01:25.299329", "seconds": 85.299329}, "exception": null, "extra": {"stage": "email_exists_prod"}, "file": {"name": "agent.py", "path": "/home/runner/work/epicfree/epicfree/epic/src/epic_games/agent.py"}, "function": "insert_challenge", "level": {"icon": "üêû", "name": "DEBUG", "no": 10}, "line": 215, "message": "Attack challenge", "module": "agent", "name": "epic_games.agent", "process": {"id": 3658, "name": "MainProcess"}, "thread": {"id": 139761568004992, "name": "MainThread"}, "time": {"repr": "2024-05-29 06:21:04.714763+08:00", "timestamp": 1716934864.714763}}}
{"text": "2024-05-29 06:21:34 | ERROR    | hcaptcha_challenger.agents.playwright.control:execute:615 | task interrupt <backcall> - {'reason': 'Failed to obtain challenge task'}\n", "record": {"elapsed": {"repr": "0:01:54.849729", "seconds": 114.849729}, "exception": null, "extra": {"reason": "Failed to obtain challenge task"}, "file": {"name": "control.py", "path": "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py"}, "function": "execute", "level": {"icon": "‚ùå", "name": "ERROR", "no": 40}, "line": 615, "message": "task interrupt <backcall>", "module": "control", "name": "hcaptcha_challenger.agents.playwright.control", "process": {"id": 3658, "name": "MainProcess"}, "thread": {"id": 139761568004992, "name": "MainThread"}, "time": {"repr": "2024-05-29 06:21:34.265163+08:00", "timestamp": 1716934894.265163}}}
{"text": "2024-05-29 06:21:34 | DEBUG    | epic_games.agent:insert_challenge:218 | Parse result - {'stage': 'email_exists_prod', 'result': <Status.CHALLENGE_BACKCALL: 'backcall'>}\n", "record": {"elapsed": {"repr": "0:01:54.850476", "seconds": 114.850476}, "exception": null, "extra": {"stage": "email_exists_prod", "result": "backcall"}, "file": {"name": "agent.py", "path": "/home/runner/work/epicfree/epicfree/epic/src/epic_games/agent.py"}, "function": "insert_challenge", "level": {"icon": "üêû", "name": "DEBUG", "no": 10}, "line": 218, "message": "Parse result", "module": "agent", "name": "epic_games.agent", "process": {"id": 3658, "name": "MainProcess"}, "thread": {"id": 139761568004992, "name": "MainThread"}, "time": {"repr": "2024-05-29 06:21:34.265910+08:00", "timestamp": 1716934894.26591}}}
{"text": "2024-05-29 06:22:05 | WARNING    | epic_games.agent:authorize:266 | ÊâßË°åË∂ÖÊó∂ - {'task': 'authorize', 'retry': 1}\n", "record": {"elapsed": {"repr": "0:02:25.933948", "seconds": 145.933948}, "exception": null, "extra": {"task": "authorize", "retry": 1}, "file": {"name": "agent.py", "path": "/home/runner/work/epicfree/epicfree/epic/src/epic_games/agent.py"}, "function": "authorize", "level": {"icon": "‚ö†Ô∏è", "name": "WARNING", "no": 30}, "line": 266, "message": "ÊâßË°åË∂ÖÊó∂", "module": "agent", "name": "epic_games.agent", "process": {"id": 3658, "name": "MainProcess"}, "thread": {"id": 139761568004992, "name": "MainThread"}, "time": {"repr": "2024-05-29 06:22:05.349382+08:00", "timestamp": 1716934925.349382}}}
{"text": "2024-05-29 06:22:08 | INFO    | epic_games.agent:_login:237 | login-with-email - {'url': 'https://www.epicgames.com/id/login?lang=en-US&noHostRedirect=true&redirectUrl=https://store.epicgames.com/en-US/free-games'}\n", "record": {"elapsed": {"repr": "0:02:28.995203", "seconds": 148.995203}, "exception": null, "extra": {"url": "https://www.epicgames.com/id/login?lang=en-US&noHostRedirect=true&redirectUrl=https://store.epicgames.com/en-US/free-games"}, "file": {"name": "agent.py", "path": "/home/runner/work/epicfree/epicfree/epic/src/epic_games/agent.py"}, "function": "_login", "level": {"icon": "‚ÑπÔ∏è", "name": "INFO", "no": 20}, "line": 237, "message": "login-with-email", "module": "agent", "name": "epic_games.agent", "process": {"id": 3658, "name": "MainProcess"}, "thread": {"id": 139761568004992, "name": "MainThread"}, "time": {"repr": "2024-05-29 06:22:08.410637+08:00", "timestamp": 1716934928.410637}}}
{"text": "2024-05-29 06:22:10 | ERROR    | hcaptcha_challenger.agents.playwright.control:handler:153 | 'utf-8' codec can't decode byte 0x9c in position 1: invalid start byte - {}\nTraceback (most recent call last):\n  File \"/home/runner/work/epicfree/epicfree/epic/src/claim.py\", line 162, in <module>\n    asyncio.run(run())\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/runners.py\", line 44, in run\n    return loop.run_until_complete(main)\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 636, in run_until_complete\n    self.run_forever()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 603, in run_forever\n    self._run_once()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 1909, in _run_once\n    handle._run()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/events.py\", line 80, in _run\n    self._context.run(self._callback, *self._args)\n> File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py\", line 145, in handler\n    data = await response.json()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/playwright/async_api/_generated.py\", line 614, in json\n    return mapping.from_maybe_impl(await self._impl_obj.json())\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/playwright/_impl/_network.py\", line 613, in json\n    return json.loads(await self.text())\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/playwright/_impl/_network.py\", line 610, in text\n    return content.decode()\nUnicodeDecodeError: 'utf-8' codec can't decode byte 0x9c in position 1: invalid start byte\n", "record": {"elapsed": {"repr": "0:02:30.676517", "seconds": 150.676517}, "exception": {"type": "UnicodeDecodeError", "value": "'utf-8' codec can't decode byte 0x9c in position 1: invalid start byte", "traceback": true}, "extra": {}, "file": {"name": "control.py", "path": "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py"}, "function": "handler", "level": {"icon": "‚ùå", "name": "ERROR", "no": 40}, "line": 153, "message": "'utf-8' codec can't decode byte 0x9c in position 1: invalid start byte", "module": "control", "name": "hcaptcha_challenger.agents.playwright.control", "process": {"id": 3658, "name": "MainProcess"}, "thread": {"id": 139761568004992, "name": "MainThread"}, "time": {"repr": "2024-05-29 06:22:10.091951+08:00", "timestamp": 1716934930.091951}}}
{"text": "2024-05-29 06:22:12 | DEBUG    | epic_games.agent:insert_challenge:215 | Attack challenge - {'stage': 'email_exists_prod'}\n", "record": {"elapsed": {"repr": "0:02:32.987424", "seconds": 152.987424}, "exception": null, "extra": {"stage": "email_exists_prod"}, "file": {"name": "agent.py", "path": "/home/runner/work/epicfree/epicfree/epic/src/epic_games/agent.py"}, "function": "insert_challenge", "level": {"icon": "üêû", "name": "DEBUG", "no": 10}, "line": 215, "message": "Attack challenge", "module": "agent", "name": "epic_games.agent", "process": {"id": 3658, "name": "MainProcess"}, "thread": {"id": 139761568004992, "name": "MainThread"}, "time": {"repr": "2024-05-29 06:22:12.402858+08:00", "timestamp": 1716934932.402858}}}
{"text": "2024-05-29 06:22:41 | ERROR    | hcaptcha_challenger.agents.playwright.control:execute:615 | task interrupt <backcall> - {'reason': 'Failed to obtain challenge task'}\n", "record": {"elapsed": {"repr": "0:03:02.538457", "seconds": 182.538457}, "exception": null, "extra": {"reason": "Failed to obtain challenge task"}, "file": {"name": "control.py", "path": "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py"}, "function": "execute", "level": {"icon": "‚ùå", "name": "ERROR", "no": 40}, "line": 615, "message": "task interrupt <backcall>", "module": "control", "name": "hcaptcha_challenger.agents.playwright.control", "process": {"id": 3658, "name": "MainProcess"}, "thread": {"id": 139761568004992, "name": "MainThread"}, "time": {"repr": "2024-05-29 06:22:41.953891+08:00", "timestamp": 1716934961.953891}}}
{"text": "2024-05-29 06:22:41 | DEBUG    | epic_games.agent:insert_challenge:218 | Parse result - {'stage': 'email_exists_prod', 'result': <Status.CHALLENGE_BACKCALL: 'backcall'>}\n", "record": {"elapsed": {"repr": "0:03:02.539082", "seconds": 182.539082}, "exception": null, "extra": {"stage": "email_exists_prod", "result": "backcall"}, "file": {"name": "agent.py", "path": "/home/runner/work/epicfree/epicfree/epic/src/epic_games/agent.py"}, "function": "insert_challenge", "level": {"icon": "üêû", "name": "DEBUG", "no": 10}, "line": 218, "message": "Parse result", "module": "agent", "name": "epic_games.agent", "process": {"id": 3658, "name": "MainProcess"}, "thread": {"id": 139761568004992, "name": "MainThread"}, "time": {"repr": "2024-05-29 06:22:41.954516+08:00", "timestamp": 1716934961.954516}}}
{"text": "2024-05-29 06:23:13 | WARNING    | epic_games.agent:authorize:266 | ÊâßË°åË∂ÖÊó∂ - {'task': 'authorize', 'retry': 2}\n", "record": {"elapsed": {"repr": "0:03:33.628776", "seconds": 213.628776}, "exception": null, "extra": {"task": "authorize", "retry": 2}, "file": {"name": "agent.py", "path": "/home/runner/work/epicfree/epicfree/epic/src/epic_games/agent.py"}, "function": "authorize", "level": {"icon": "‚ö†Ô∏è", "name": "WARNING", "no": 30}, "line": 266, "message": "ÊâßË°åË∂ÖÊó∂", "module": "agent", "name": "epic_games.agent", "process": {"id": 3658, "name": "MainProcess"}, "thread": {"id": 139761568004992, "name": "MainThread"}, "time": {"repr": "2024-05-29 06:23:13.044210+08:00", "timestamp": 1716934993.04421}}}
{"text": "2024-05-29 06:23:13 | ERROR    | __main__:run:158 | An error has been caught in function 'run', process 'MainProcess' (3658), thread 'MainThread' (139761568004992): - {}\nTraceback (most recent call last):\n  File \"/home/runner/work/epicfree/epicfree/epic/src/claim.py\", line 162, in <module>\n    asyncio.run(run())\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/runners.py\", line 44, in run\n    return loop.run_until_complete(main)\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 636, in run_until_complete\n    self.run_forever()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 603, in run_forever\n    self._run_once()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 1909, in _run_once\n    handle._run()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/events.py\", line 80, in _run\n    self._context.run(self._callback, *self._args)\n> File \"/home/runner/work/epicfree/epicfree/epic/src/claim.py\", line 158, in run\n    await agent.stash()\n  File \"/home/runner/work/epicfree/epicfree/epic/src/claim.py\", line 151, in stash\n    await self.claim_epic_games(context)\n  File \"/home/runner/work/epicfree/epicfree/epic/src/claim.py\", line 98, in claim_epic_games\n    if await epic.authorize(page):\n  File \"/home/runner/work/epicfree/epicfree/epic/src/epic_games/agent.py\", line 269, in authorize\n    raise RuntimeError(f\"Failed to flush token - agent={self.__class__.__name__}\")\nRuntimeError: Failed to flush token - agent=EpicGames\n", "record": {"elapsed": {"repr": "0:03:33.804619", "seconds": 213.804619}, "exception": {"type": "RuntimeError", "value": "Failed to flush token - agent=EpicGames", "traceback": true}, "extra": {}, "file": {"name": "claim.py", "path": "/home/runner/work/epicfree/epicfree/epic/src/claim.py"}, "function": "run", "level": {"icon": "‚ùå", "name": "ERROR", "no": 40}, "line": 158, "message": "An error has been caught in function 'run', process 'MainProcess' (3658), thread 'MainThread' (139761568004992):", "module": "claim", "name": "__main__", "process": {"id": 3658, "name": "MainProcess"}, "thread": {"id": 139761568004992, "name": "MainThread"}, "time": {"repr": "2024-05-29 06:23:13.220053+08:00", "timestamp": 1716934993.220053}}}
